pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -249,6 +290,16 @@ def _get_provider(
     def backend_info(self) -> BackendInfo:
         return self._backend_info
 
+    @property
```

Comment:
Here I've used the syntax recommended in https://stackoverflow.com/questions/30585427/raise-an-exception-if-class-attribute-is-set-to-a-certain-value-after-instantiat?rq=3

Let me know if this is acceptable